app/services/create_project_activity.rb

```diff
@@ -13,7 +13,13 @@ def call
 
     activity = Activity.new
     activity.organisation = reporting_organisation
-    activity.reporting_organisation = reporting_organisation
+
+    if reporting_organisation.is_government_organisation?
```

Comment:
This looks like it does the trick 👍 

I noticed that there's some related code that uses `reporting_organisation` for setting the exetending organsiation association too. It feels like we could make the pair a little clearer?

I wondered if the use of private methods could help break this down to make it more readable, what do you think?

```
private 
def creating_organisation
  Organisation.find(organisation_id)

def service_owner_organisation
  Organisation.find_by(service_owner: true)

def extending_organisation
  creating_organisation

def reporting_organisation
  if managing_organisation.is_government_organisation?
    service_owner_organisation
  else 
    managing_organisation
```